docs/data-sources/mapreduce_clusters.md

```diff
@@ -0,0 +1,298 @@
+
+* `availability_zone` - The AZ.
+
+* `vnc` - URI for remotely logging in to an ECS.
+
+* `volume_size` - Disk storage space.
+
+* `volume_type` - Disk type.
+
+* `enterprise_project_id` - Enterprise project ID.
+
+* `type` - Cluster type.  
+  The following options are supported:  
+    + **0**: analysis cluster.
+    + **1**: streaming cluster.
+    + **2**: hybrid cluster.
+    + **3**: custom cluster.
+    + **4**: Offline cluster.
+
+* `security_group_id` - Security group ID.
+
+* `slave_security_group_id` - Security group ID of a non-Master node.  
+  Currently, one MRS cluster uses only one security group. Therefore, this field has been discarded.
+
+* `stage_desc` - Cluster progress description.  
+  The cluster installation progress includes:
+    + **Verifying cluster parameters**: Cluster parameters are being verified.
+    + **Applying for cluster resources**: Cluster resources are being applied for.
+    + **Creating VMs**: The VMs are being created.
+    + **Initializing VMs**: The VMs are being initialized.
+    + **Installing MRS Manager**: MRS Manager is being installed.
+    + **Deploying the cluster**: The cluster is being deployed.
+    + **Cluster installation failed**: Failed to install the cluster.
+
+  The cluster scale-out progress includes:
+    + **Preparing for scale-out**: Cluster scale-out is being prepared.
+    + **Creating VMs**: The VMs are being created.
+    + **Initializing VMs**: The VMs are being initialized.
+    + **Adding nodes to the cluster**: The nodes are being added to the cluster.
+    + **Scale-out failed**: Failed to scale out the cluster.
+
+  The cluster scale-in progress includes:
+    + **Preparing for scale-in**: Cluster scale-in is being prepared.
+    + **Decommissioning instance**: The instance is being decommissioned.
+    + **Deleting VMs**: The VMs are being deleted.
+    + **Deleting nodes from the cluster**: The nodes are being deleted from the cluster.
+    + **Scale-in failed**: Failed to scale in the cluster.
+
+  If the cluster installation, scale-out, or scale-in fails, stageDesc will display the failure cause.
+
+* `safe_mode` - Running mode of an MRS cluster.  
+  The following options are supported:
+    + **0**: Normal cluster.
+    + **1**: Security cluster.
+
+* `version` - Cluster version.
+
+* `node_public_cert_name` - Name of the key file.
+
+* `master_node_ip` - IP address of a Master node.
+
+* `private_ip_first` - Preferred private IP address.
+
+* `tags` - The tag information.
+
+* `log_collection` - Whether to collect logs when cluster installation fails.  
+  The following options are supported:
+    + **0**: Do not collect logs.
+    + **1**: Collect logs.
+
+* `task_node_groups` - List of Task nodes.
+  The [NodeGroup](#MrsClusters_ClustersNodeGroup) structure is documented below.
+
+* `node_groups` - List of Master, Core and Task nodes.
+  The [NodeGroup](#MrsClusters_ClustersNodeGroup) structure is documented below.
```

Comment:
same in these two rows, pls check

Reply:
When a struct is uesd by two or more fields, all the fields should use the consistent structure name,block name and anchor point